### PR TITLE
Changed  mirror for eclipse download

### PR DESCRIPTION
**Pull request by sujith-mn**

### CODE REVIEW

**Comment by SchettlerKoehler** [comment]
For me, the mirror is very slow (>20min for 300MB). Please can someone else also check how fast this download is: http://ftp.kaist.ac.kr/eclipse/technology/epp/downloads/release/2021-03/R/eclipse-java-2021-03-R-win32-x86_64.zip

**Comment by maybeec** [comment]
For me it was really fast: 2MB from inside the office. Did you check it without or just with VPN @SchettlerKoehler ?

File: scripts/src/main/resources/scripts/command/eclipse [hunk 1]
```diff
@@ -41,22 +41,43 @@ function doInstallEclipsePlugin() {
...
-    then
-      download_url="${download_url}-macosx-cocoa-x86_64.dmg"
-    elif doIsWindows
-    then
-      download_url="${download_url}-win32-x86_64.zip"
-    else
-      download_url="${download_url}-linux-gtk-x86_64.tar.gz"
+    doPrepareDownloadUrl 
+    if [ -z "$download_url" ]; then 
+     doFail "Eclipse binary is not available "
```
Comment (SchettlerKoehler) [inline scripts/src/main/resources/scripts/command/eclipse:80]: Please improve the error message. I would suggest something like 'The eclipse version was not found on any of the mirrors. Please create an issue on https://github.com/devonfw/ide/issues'

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: scripts/src/main/resources/scripts/command/eclipse (hunk 1 above)
Comment (sujith-mn) [reply scripts/src/main/resources/scripts/command/eclipse]: Changed the message.

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**